management/src/test/java/org/apache/tamaya/management/internal/ManagedConfigTest.java

```diff
@@ -75,19 +72,14 @@ public void testGetConfigurationArea() {
     @Test
     public void testGetAreas() {
         Set<String> sections = (bean.getSections());
-        assertThat(sections).isNotNull();
-        assertThat(sections.contains("java")).isTrue();
-        assertThat(sections.contains("file")).isTrue();
+        assertThat(sections).isNotNull().contains("java", "file");
     }
 
     @Test
     public void testGetTransitiveAreas() {
         Set<String> sections = (bean.getTransitiveSections());
         Set<String> sectionsNT = (bean.getSections());
-        assertThat(sections).isNotNull();
-        assertThat(sections.contains("java")).isTrue();
-        assertThat(sections.contains("sun")).isTrue();
-        assertThat(sections.contains("sun.os")).isTrue();
+        assertThat(sections).isNotNull().contains("java", "sun", "sun.os");
         assertThat(sectionsNT.size() < sections.size()).isTrue();
```

Comment:
Just found out that it'S even simpler:
assertThat(sections.size()).isGreaterThan(sections.size());

Thanks

Reply:
@ottlinger that's an excellent suggestion. I just added a commit to make use of `isGreaterThan` (and related methods).